### PR TITLE
Move changelogs to yankee

### PR DESCRIPTION
**Pull request by vikiCoder**

### CODE REVIEW

File: CHANGELOG.yaml [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+    - Move vendor out one level
+
+0.0.1-beta.2:
+  date: 2016-11-30
+  new features:
+    - Made the dependency on bundling not be needed if cache is present
+    - Added pre-publish script for initial testing
+
+0.0.1-beta.1:
+  date: 2016-11-xy
```
Comment (vikiCoder) [inline CHANGELOG.yaml]: Date missing here...

Comment (codenirvana) [reply CHANGELOG.yaml]: `27 Dec 2016`
https://github.com/postmanlabs/postman-sandbox/releases?after=v2.1.0-beta.1

File: CHANGELOG.yaml [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+      Updated `chai-postman` to `v1.1.1`, which fixes a JSON schema assertion
+      bug
+
+3.2.0:
+  date: 2018-12-24
+  new features:
+    - Added Ajv JSON schema validator
+    - Deprecated tv4 in favor of Ajv
+  fixed bugs:
+    - Fixed a bug that caused negated assertions to persist
```
Comment (codenirvana) [inline CHANGELOG.yaml]: GH-374

File: CHANGELOG.yaml [hunk 3]
```diff
@@ -0,0 +1,361 @@
...
+  date: 2018-12-25
+  fixed bugs:
+    - >-
+      Updated `chai-postman` to `v1.1.1`, which fixes a JSON schema assertion
+      bug
+
+3.2.0:
+  date: 2018-12-24
+  new features:
+    - Added Ajv JSON schema validator
```
Comment (codenirvana) [inline CHANGELOG.yaml]: GH-373

File: CHANGELOG.yaml [hunk 4]
```diff
@@ -0,0 +1,361 @@
...
+    - Deprecated tv4 in favor of Ajv
+  fixed bugs:
+    - Fixed a bug that caused negated assertions to persist
+  chores:
+    - Updated dependencies
+    - Migrated tests to chai expect assertions
```
Comment (codenirvana) [inline CHANGELOG.yaml]: Move this above `Updated dependencies`

**Pull request merged**